src/cdk/testing/tests/webdriver.e2e.spec.ts

```diff
@@ -3,34 +3,32 @@ import {
   SeleniumWebDriverHarnessEnvironment,
   waitForAngularReady,
 } from '../../testing/selenium-webdriver';
-import {runfiles} from '@bazel/runfiles';
 import * as webdriver from 'selenium-webdriver';
 import {crossEnvironmentSpecs} from './cross-environment.spec';
 import {MainComponentHarness} from './harnesses/main-component-harness';
+import {Options, setDefaultService, ServiceBuilder} from 'selenium-webdriver/chrome';
+import path from 'path';
 
 // Tests are flaky on CI unless we increase the timeout.
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 10_000; // 10 seconds
 
-/**
- * Metadata file generated by `rules_webtesting` for browser tests.
- * The metadata provides configuration for launching the browser and
- * necessary capabilities. See source for details:
- * https://github.com/bazelbuild/rules_webtesting/blob/06023bb3/web/internal/metadata.bzl#L69-L82
- */
-interface WebTestMetadata {
-  capabilities: webdriver.Capabilities;
-}
-
-if (process.env['WEB_TEST_METADATA'] === undefined) {
+if (process.env['TEST_SERVER_PORT'] === undefined) {
   console.error(`Test running outside of a "web_test" target. No browser found.`);
   process.exit(1);
 }
 
-const webTestMetadata: WebTestMetadata = require(
-  runfiles.resolve(process.env['WEB_TEST_METADATA']),
-);
+const projectRoot = path.resolve('../../../');
```

Comment:
This is a magic string for the project root? Can we find this in a more portable way?

Reply:
It's not magic because it's just relative from the current file; which is now possible with `rules_js`; but we should make it more clear and use `dirname`.